arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -0,0 +1,763 @@
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "VaultExtendedInfo"
+        ],
+        "description": "Update vault extended info.",
+        "operationId": "VaultExtendedInfo_Update",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceResourceExtendedInfoDetails",
+            "in": "body",
+            "description": "resourceResourceExtendedInfoDetails",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "ClientDiscoveryDisplay": {
+      "description": "Localized display information of an operation.",
+      "type": "object",
+      "properties": {
+        "Provider": {
+          "description": "Name of the resource provider.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Resource": {
+          "description": "Type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Operation": {
+          "description": "Name of the operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Description": {
+          "description": "Operation description.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryForLogSpecification": {
+      "description": "Log specification for the operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the log.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Localized display name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "blobDuration": {
+          "description": "Blob duration.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryForProperties": {
```

Comment:
Do you mean to say that this is repeated across multiple JSONs? Is there a way to share definitions across swagger json files?